RX.h

```diff
@@ -9,6 +9,11 @@ uint32_t lastPacketTimeUs = 0;
 uint32_t lastRSSITimeUs = 0;
 uint32_t linkLossTimeMs;
 
+uint32_t hopInterval = 0;
+uint32_t hopTimeout = 0;
+uint32_t hopTimeoutSlow = 0;
+uint32_t RSSI_timeout = 0;
```

Comment:
My hunch would be to write it `rssiTimeout`, which seems to be most common way to deal with abbreviations. In general, I don't like the mix of camel case and underscores, as it makes remembering what uses camel case and / or underscores hard.